carLot/CarLot.js

```diff
@@ -10,6 +12,26 @@ var CarLot = (function (carLot) {
   })
   xhr.send();
 
+// Adding public function that exposes inventory
+
+  carLot.loadInventory = function (inventory) {
```

Comment:
With this function you will not need the top portion as this will make your xhr call. Also, you should be passing your populate dom function into this one so you can use it on you populate page. Take a look at my repo and compare the two files. I call loadInventory from my quiz.js. I pass my populatePage function into it so that inventory is available to the populatePage function. https://github.com/gregkorte/Cohort12/tree/master/FrontEnd/single-page-applications/q_spa